kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties

```diff
@@ -0,0 +1,9 @@
+BaseStepDialog.StepName=Step name:
```

Comment:
I do not remember exactly where it was. I will merge and then I am sure QA will test on all OSs. Maybe he had issues where it showed on Linux, but not on windows or mac. I'll merge either way. Thx for the comments :)